bin/tests/bootstrap-unit.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/sbin/bats
+
+load 'test_helper/bats-support/load'
+load 'test_helper/bats-assert/load'
+load 'test_helper/bats-file/load'
+
+setup () {
+    TEST_TEMP_DIR="$(temp_make --prefix 'otomi-values-')"
+    export ENV_DIR="$TEST_TEMP_DIR"
+    export ENV_PATH="$ENV_DIR/env"
```

Comment:
ENV_PATH does not need to be exported, does it?

Reply:
I assume it does because all the child processes need access to it?